tests/functional/pages/surveys/household_composition/summary.page.js

```diff
@@ -2,18 +2,23 @@ import SummaryPage from '../../summary.page'
 
 class HouseholdCompositionSummary extends SummaryPage {
 
-  clickEdit() {
-    browser.click('[data-qa="household-full-name-edit"]')
+  clickAddAnother() {
+    browser.click('[name="household-composition-add-another"]')
+    return this
+  }
+
+  submit() {
+    browser.click('.qa-btn-submit')
     return this
   }
 
   isNameDisplayed(name) {
-    return browser.element('[data-qa="household-full-name-answer"]').getText() === name
+    return browser.element('.box--entity__person').getText() === name
```

Comment:
Yeah, this is subject to whatever we decide the template we're rendering is going to end up.